src/signals/incident-management/containers/Dashboard/hooks/useDepartments.ts

```diff
@@ -35,7 +36,10 @@ export const useDepartments = (): {
           (code, value) => code + value,
           ''
         )
-        if (!cachedDepartments[departmentCodes]) {
+        if (
```

Comment:
A test should be updated to support this change right?

Reply:
is uses the default config, I could make a test that its actually not working due to another config, you prefer?

Reply:
added a test